doc/dev/TESTING.md

```diff
@@ -1,77 +1,110 @@
 # Testing
 
-F3D has an expansive suite of tests, you may want to run them locally,
-either to validate your build or because you are contributing to F3D and want to add/modify a test.
+F3D has an expansive suite of tests that can be run locally, either to validate your build or because you are contributing to F3D and want to add/modify a test.
```

Comment:
Did we mean "extensive" here?
```suggestion
F3D has an extensive suite of tests that can be run locally, either to validate your build or because you are contributing to F3D and want to add/modify a test.
```